tests/DebtLocker.t.sol

```diff
@@ -1133,6 +1132,22 @@ contract DebtLockerV4Migration is TestUtils {
         assertEq(loan.pendingLender(), newLender);
     }
 
+    function test_acl_acceptLender() external {
+        ( MapleLoan loan, DebtLocker debtLocker ) = _createFundAndDrawdownLoan(1_000_000, 30_000);
+
+        LoanMigrator loanMigrator    = new LoanMigrator();
+        LoanMigrator notLoanMigrator = new LoanMigrator();
+
+        address newLender = address(3);
+
+        poolDelegate.debtLocker_upgrade(address(debtLocker), 2, abi.encode(address(loanMigrator)));
+
+        loanMigrator.debtLocker_setPendingLender(address(debtLocker), address(debtLocker));  // Set pending lender in Loan to get ACL to work
+
+        assertTrue(!notLoanMigrator.try_debtLocker_acceptLender(address(debtLocker)));
```

Comment:
Think we should assert with the correct error msg as it would have highlighted the incorrect func selector but appreciate this means adjust the `LoanMigrator.sol` in accounts